Windows version missing dependency when importing transactions

OS: Windows 10
Nano Version 1.3.0-rc1

### Expected behaviour

Imported votes do not throw a missing dependency when trying to generate a transaction.

### Actual behaviour

![image](https://user-images.githubusercontent.com/16565680/33240698-77093b32-d2ba-11e7-956e-36dae60af39b.png)

```
[Window Title]
Error

[Main Instruction]
A JavaScript error occurred in the main process

[Content]
Uncaught Exception:
Error: Cannot find module '../screen'
    at Module._resolveFilename (module.js:485:15)
    at Function.Module._resolveFilename (C:\Users\Isabella\AppData\Local\Programs\lisk-nano\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.get [as screen] (C:\Users\Isabella\AppData\Local\Programs\lisk-nano\resources\electron.asar\browser\api\exports\electron.js:11:16)
    at Object.init (C:\Users\Isabella\AppData\Local\Programs\lisk-nano\resources\app.asar\build\main.js:27138:41)
    at Object.create (C:\Users\Isabella\AppData\Local\Programs\lisk-nano\resources\app.asar\build\main.js:27173:9)
    at App.<anonymous> (C:\Users\Isabella\AppData\Local\Programs\lisk-nano\resources\app.asar\build\main.js:27528:17)
    at emitTwo (events.js:130:20)
    at App.emit (events.js:213:7)

[OK]
```
### Steps to reproduce

Navigate to lisk.builders and create a vote list.

Initiate vote with the first step that takes you into lisk nano to sign the transaction

### Soution:

run `npm update` on the windows build machine and rebuild the binaries.

--- a/app/src/main.js
+++ b/app/src/main.js
@@ -48,7 +48,7 @@ const isSecondInstance = app.makeSingleInstance((argv) => {
 });
 
 if (isSecondInstance) {
-  app.quit();
+  app.exit();
 }
 
 app.on('will-finish-launching', () => {

